qcodes/dataset/descriptions.py

```diff
@@ -0,0 +1,67 @@
+import io
+from typing import Dict, Any
+
+from ruamel.yaml import YAML
```

Comment:
we might need to handle that the conda package is `ruamel_yaml` e.g. catch the import error and try with this as an alternative.